app/assets/stylesheets/style.css

```diff
@@ -640,3 +640,9 @@ textarea, input {
 {
     color: #f1f1f1 !important;
 }
+
+@media (max-width: 992px) {
+  .hidden-sidebar {
+    display: none !important;
+  }
+}
```

Comment:
@ShivamJhaa Instead of adding an extra css style, we can go with your suggestion and change `.hidden-sm, .hidden-xs` to `.d-sm-none, .d-xs-none`. This way we keep the changes to only the `sidebar.js` file.

Reply:
@ShivamJhaa This file is still in this PR. Make sure you undo all of the changes before pushing.

Reply:
Hi @ShivamJhaa, just wanted to check-in. We're happy to help if you need any assistance with the changes requested.